app/javascript/packages/phone-input/index.spec.ts

```diff
@@ -102,18 +103,20 @@ describe('PhoneInput', () => {
     expect(input.querySelector('.iti.iti--allow-dropdown')).to.be.ok();
   });
 
-  it('validates input', async () => {
+  it('validates input for US phone numbers', async () => {
     const input = createAndConnectElement();
 
     const phoneNumber = getByLabelText(input, 'Phone number') as HTMLInputElement;
 
     expect(phoneNumber.validity.valueMissing).to.be.true();
 
-    await userEvent.type(phoneNumber, '5');
-    expect(phoneNumber.validationMessage).to.equal('Phone number is not valid');
+    await userEvent.type(phoneNumber, '5555');
+    expect(phoneNumber.validationMessage).to.equal(
+      'Enter a 10 digit phone number. Example: (201) 555-0123',
+    );
 
     await userEvent.type(phoneNumber, '13-555-1234');
-    expect(phoneNumber.validity.valid).to.be.true();
+    expect(phoneNumber.validity.valid).to.be.false();
```

Comment:
The intent of this assertion was to verify that it becomes valid again after entering a complete, valid number.

Could we keep the initial behavior where it validates against a single digit, and then the full number? Since I would expect entering just `'5'` should have the same behavior.